### ISSUE TITLE
Skin Components by Skin Version

### ISSUE THREAD

**Issue by baconpaul** [surge-synthesizer/surge#3415]
So in 1.9 we are going to add a bunch of controls and components and skin em up. Cool
But people may have custom 1.8 skins.
Luckily for us, I made the skin engine have a version from the outset. Right now it is just required to be 1.
But for 1.9

1. Lets up it to 2
2. Lets have elements in SkinModel.cpp have an 'asOfSkinVersion' property 
3. If SGE is laying out an element and the skin version is < the component asOf, it ignores it

This means that old skins could continue to work just fine when we add a new slider filter (I mean, they won't have the slider filter displayed of course) 

We could also do this on things like positions. Like have a control default XYWH be a function of skin version.

Basically: Make SkinModel version aware and make SGE version aware so we can keep old skins running, even if incomplete, as we expand in 19
